test

```diff
@@ -185,16 +188,12 @@ def test_all(year, event, problem, language, id):
 
   checker = prepare_checker(language, file)
 
-  results = [
-    test(checker, input, expected)
-    for input, expected in test_case
-  ]
+  for input, expected in test_case:
+    yield test(checker, input, expected)
```

Comment:
I'll check this code in my own machine. Maybe the github editor can be give me a horrible impression about that. If be how I was thinking that is, I really don't understand how this could be compiled.